bazel/configure_template.bzl

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Define a rule to configure .in template files.
+
+We use custom *.BUILD files for two external dependencies that do not natively support Bazel: crc32c and libcurl.
+Both require configuring a `.in` file, this creates a rules to perform this configuration.
```

Comment:
s/file, this/file. This/
s/a rules/a rule/

Reply:
Fixed.